reducers/views/channel_sidebar.ts

```diff
@@ -75,8 +75,62 @@ export function newCategoryIds(state: string[] = [], action: GenericAction): str
     }
 }
 
+export function selectedChannelIds(state: string[] = [], action: GenericAction): string[] {
```

Comment:
0/5, but we're running into a bit of a naming conflict here since we have a concept of a "selected post/channel" which are for the post visible in the RHS similar to the actions. `multiselectedChannelIds` might be a better name if this is only used when actually multiselecting

Reply:
Yeah I think that makes sense. Will change.